rero_ils/modules/locations/api.py

```diff
@@ -68,6 +68,15 @@ def location_pids(self, library_pid, source='pid'):
         return [location.pid for location in self.filter(
                 'term', library__pid=library_pid).source(source).scan()]
 
+    def by_organisation_id(self, organisation_pid):
```

Comment:
Better to rename  to `by_organisation_pid`. Normally id is not the same thing as pid.